### PR TITLE
Update whisper_speech_recognition.py

### PR DESCRIPTION
**Pull request by raymondlo84**
increased the max_new_tokens and also added device to show how to use GPU. Also added language options example as a comment

### CODE REVIEW

File: samples/python/whisper_speech_recognition/whisper_speech_recognition.py [hunk 1]
```diff
@@ -6,12 +6,10 @@
 import openvino_genai
 import librosa
 
-
 def read_wav(filepath):
     raw_speech, samplerate = librosa.load(filepath, sr=16000)
     return raw_speech.tolist()
 
-
```
Comment (Wovchena) [inline samples/python/whisper_speech_recognition/whisper_speech_recognition.py:14]: Having two lines between functions is a common code style. See PEP8. Please, revert it.

File: samples/python/whisper_speech_recognition/whisper_speech_recognition.py [hunk 2]
```diff
@@ -20,27 +18,26 @@ def main():
 
     raw_speech = read_wav(args.wav_file_path)
 
-    pipe = openvino_genai.WhisperPipeline(args.model_dir)
+    device = "CPU" # can switch between CPU/GPU
+    pipe = openvino_genai.WhisperPipeline(args.model_dir, device=device)
```
Comment (Wovchena) [inline samples/python/whisper_speech_recognition/whisper_speech_recognition.py:22]: Can you update C++ version in the same way to keep them aligned?

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: samples/python/whisper_speech_recognition/whisper_speech_recognition.py (hunk 1 above)
Comment (raymondlo84) [reply samples/python/whisper_speech_recognition/whisper_speech_recognition.py:14]: Got it.

File: samples/python/whisper_speech_recognition/whisper_speech_recognition.py (hunk 2 above)
Comment (raymondlo84) [reply samples/python/whisper_speech_recognition/whisper_speech_recognition.py:24]: I have to spin up the C++ and update after. Maybe we will merge this first, and will work on C++ on the next PR?

File: samples/python/whisper_speech_recognition/whisper_speech_recognition.py [hunk 3]
```diff
@@ -20,24 +20,25 @@ def main():
...
+        max_new_tokens=1000, #increase this based on your speech length
         # 'task' and 'language' parameters are supported for multilingual models only
-        language="<|en|>",
+        language="<|en|>", #can switch to <|zh|> for Chinese language 
         task="transcribe",
         return_timestamps=True,
         streamer=streamer,
     )
-
+
```
Comment (Wovchena) [inline samples/python/whisper_speech_recognition/whisper_speech_recognition.py:39]:
```suggestion

```

File: samples/python/whisper_speech_recognition/whisper_speech_recognition.py [hunk 4]
```diff
@@ -20,24 +20,25 @@ def main():
...
+        language="<|en|>", #can switch to <|zh|> for Chinese language 
         task="transcribe",
         return_timestamps=True,
         streamer=streamer,
     )
-
+    
     print()
-
+
```
Comment (Wovchena) [inline samples/python/whisper_speech_recognition/whisper_speech_recognition.py:41]:
```suggestion

```

**Review by Wovchena — DISMISSED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

File: samples/python/whisper_speech_recognition/whisper_speech_recognition.py [hunk 5]
```diff
@@ -21,9 +21,9 @@ def main():
     config = openvino_genai.WhisperGenerationConfig(
         args.model_dir + "/generation_config.json"
     )
-    config.max_new_tokens = 100
+    config.max_new_tokens = 100 # increase this based on your speech length
```
Comment (ilya-lavrenov) [inline samples/python/whisper_speech_recognition/whisper_speech_recognition.py:24]:
```suggestion
    config.max_new_tokens = 500 # increase this based on your speech length
```

File: samples/python/whisper_speech_recognition/whisper_speech_recognition.py [hunk 6]
```diff
@@ -21,9 +21,9 @@ def main():
     config = openvino_genai.WhisperGenerationConfig(
         args.model_dir + "/generation_config.json"
     )
-    config.max_new_tokens = 100
+    config.max_new_tokens = 500 # increase this based on your speech length
```
Comment (ilya-lavrenov) [inline samples/python/whisper_speech_recognition/whisper_speech_recognition.py:24]:
```suggestion
    config.max_new_tokens = 100 # increase this based on your speech length
```

**Review by as-suvorov — APPROVED** [review]

**Review by Wovchena — APPROVED** [review]
build_jenkins

**Comment by Wovchena** [comment]
build_jenkins

**Pull request merged**